Products/PerFactErrors/errors.py

```diff
@@ -50,7 +51,53 @@ def afterfail_error_message(event):
         if body is not None:
             req.response.setBody(body)
     except Exception:
+        logger.warn(traceback.format_exc())
```

Comment:
Why is this here? `logger.exception` in the next line already includes the current traceback.

Reply:
I checked the traceback in the result again. I will drop this line